lib/paginate.js

```diff
@@ -2,37 +2,33 @@
 
 class Paginate {
   constructor(resource, args) {
-    let _this = this;
-
-    _this.resource = resource;
-    _this.response = args.body;
-    _this.options  = args.options;
-    _this.page     = _this.options.page ? _this.options.page : 1;
-    _this.method   = args.method;
-    _this.last     = Math.ceil(args.body.total / (args.body.count * _this.page));
+    this.resource = resource;
```

Comment:
The pagination methods need to be rebuilt, but it's not necessary for this current version. These aren't being used anywhere, so I decided to just leave it as-is for now (since it's not really affecting anything anyway).